tests/__init__.py

```diff
@@ -46,5 +47,5 @@ def assertDeepAlmostEqual(self, expected, actual, *args, **kwargs):
             exc.__dict__.setdefault('traces', []).append(trace)
             if is_root:
                 trace = ' -> '.join(reversed(exc.traces))
-                exc = AssertionError("%s\nTRACE: %s" % (exc.message, trace))
+                exc = AssertionError("%s\nTRACE: %s" % (str(exc), trace))
```

Comment:
Just a note: this function seems like a copy of another one in Rupy/ExPrESS - a candidate to be moved to a utils package

Reply:
Yea, for sure, this one's been on my hit list for consolidation for a while.

Reply:
The particular bug here was noteworthy though - turns out exception.message was deprecated in python 2.6, and removed in python 3.  I had never seen it before, so it got past me at first on the migration.
So going forward, the right move for getting the message out of an exception is either explicitly or implicitly casting it to a string.